### DISCUSSION TITLE
Setup step 3 help: wait for 1st build of site

### DISCUSSION THREAD

**Discussion by rebullm** [greenelab/lab-website-template#189] [General] [answered]
Hi- I'm on step 3, and believe I have changed the settings to build/publish from gh-pages, but I am not sure how to trigger the first build to occur. 

In your repo's "⚙️ Settings", [set GitHub Pages to build/publish from](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site) the gh-pages branch. Wait for the first build of your site to complete (~3 minutes, multiple Actions workflows will run).

Thanks, 
Meg

**Comment by rebullm** [comment]
I also did step 4, but still don't see a step 5 readme file...

**Comment by vincerubinetti** [comment, marked as answer]
When I [go to your repo](https://github.com/rebullm/Bennett-Lab-website-v1.1/), it shows me the `gh-pages` branch, which I think means you set that branch as the default branch for the repo. The default branch should be `main`. 

The GitHub Pages branch should be set to `gh-pages`. Can you take a screenshot of your Settings -> Pages screen and paste it in a comment here. It should look something like this:

<img width="912" alt="Screenshot 2023-04-25 at 3 00 04 PM" src="https://user-images.githubusercontent.com/8326331/234376470-e4dcb3f3-22df-4d5a-8992-6f085589d567.png">

The first build of your site (and also the link in the readme) occur automatically from step 3. I see in your [actions](https://github.com/rebullm/Bennett-Lab-website-v1.1/actions) no build of the site has happened, and that's probably because you haven't enabled GitHub Pages.

My suspicion is that you set the repo's _default branch_ to `gh-pages` instead of the Pages build-from branch. Please see the linked instructions in the step:
https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site

**Comment by rebullm** [comment]
okay- that was it- I wasn't looking in the correct place for github pages. I've corrected my mistake and made the change. But still don't have a page yet....any ideas? 
![image](https://user-images.githubusercontent.com/93389344/234378790-a288640e-9bfb-48ed-b112-de77313bd5dc.png)

**Comment by rebullm** [comment]
wait a minute! I do! thanks!
